src/Forms/Validation.jsx

```diff
@@ -39,9 +36,11 @@ export default class Validation extends Component {
           (prev, error) => ({ ...prev, [error.field]: error.message }),
           {}
         );
+        isValid = false;
       }
       this.setState({ errors: err });
     });
+    return isValid;
```

Comment:
so instead of doing a boolean does the result from the validator package not return false? If not, we could also just return the errors and use that to check if there are errors or not.

Reply:
yes we can return the errors and when there are no errors then `validator` package return null so this will work indeed.